### ISSUE TITLE
Display the list of attachments in a page

### ISSUE THREAD

**Issue by Diorella90** [elinsoftware/service-portal-attachments#2]
Could you please provide the steps you did to create the page to display the list of attachments? I can't get that part. Do you have to change the sys_id every time you want to see attachments?

Also, how to add the widget to a form. I use a Catalog Task form.

Thanks for your help.
Diorella.

**Comment by andrew-psch** [comment]
The article references a [Service Portal page](https://docs.servicenow.com/bundle/quebec-servicenow-platform/page/build/service-portal/concept/c_Pages.html), so you can create your own or use the out of the box pages.

**Comment by Diorella90** [comment]
Thanks. I was able to create the page and display the widget. How did you load the service portal page to the form (incident)? I see you have a tab.

![image](https://user-images.githubusercontent.com/81320390/112390163-7daa0700-8cbb-11eb-809f-6df360c8bd93.png)

**Comment by andrew-psch** [comment]
Yes, you just need to create a custom UI Macro/Formatter and use iFrame to load a Service Portal page. There is a section in [this blog post](https://pishchulin.medium.com/signature-pad-for-servicenow-applications-ab3c4677e4e1) that explains the process.

**Comment by Diorella90** [comment]
Thank you Andrew. I followed the process but I'm still having issues to display the Formatter in the form. This is what I did:

1. I created a service portal called My Page Portal. URL suffix is mpp. I just added that. Did I miss something from there? See the picture. When I did click on try it works. The src line works fine I tested it.

![image](https://user-images.githubusercontent.com/81320390/113784118-18f69f80-96f2-11eb-8d55-cf0f31956399.png)

2. I created the UI Macro and do it in this way. I had to add the tr and td tags. This [article ](https://docs.servicenow.com/bundle/orlando-platform-administration/page/administer/form-administration/task/t_CreateAUIMacro.html)says that. 

![image](https://user-images.githubusercontent.com/81320390/113784237-52c7a600-96f2-11eb-9377-d19d2cdb081b.png)

3. Then I created the Formatter. 
![image](https://user-images.githubusercontent.com/81320390/113784467-ba7df100-96f2-11eb-83db-3e45a1572810.png)

4. When I added the formatter to the form it does not display anything. 

![image](https://user-images.githubusercontent.com/81320390/113784521-d4b7cf00-96f2-11eb-80a8-c3048b104085.png)

Could you please advise? I'm almost done with this and I really appreciate any help!

**Comment by andrew-psch** [comment]
@Diorella90 send me an email to ap@elinsoftware.com

**Comment by Diorella90** [comment]
I just sent you an email.
